frontend/yarn.lock

```diff
@@ -14854,14 +14854,7 @@ npmlog@^5.0.1:
     gauge "^3.0.0"
     set-blocking "^2.0.0"
 
-nth-check@^1.0.2, nth-check@~1.0.1:
-  version "1.0.2"
-  resolved "https://registry.yarnpkg.com/nth-check/-/nth-check-1.0.2.tgz#b2bd295c37e3dd58a3bf0700376663ba4d9cf05c"
-  integrity sha512-WeBOdju8SnzPN5vTUJYxYUxLeXpCaVP5i5e0LF8fg7WORF2Wd7wFX/pk0tYZk7s8T+J7VLy0Da6J1+wCT0AtHg==
-  dependencies:
-    boolbase "~1.0.0"
-
-nth-check@^2.0.1:
+nth-check@2.1.1, nth-check@^1.0.2, nth-check@^2.0.1, nth-check@~1.0.1:
```

Comment:
when I pull the branch and run yarn install in frontend, I get a couple warnings

 ```
warning Resolution field "nth-check@2.1.1" is incompatible with requested version "nth-check@^1.0.2"
warning Resolution field "nth-check@2.1.1" is incompatible with requested version "nth-check@~1.0.1"
```

Reply:
I appreciate you calling this out! We've got _lots_ of various warnings for this sort of thing, unfortunately...these will all hopefully be rectified as we roll up new major versions of these tools, but this hopefully should not affect functionality for now!